string vendor_name
string product_name
string serial_number
string article_number
